index.js

```diff
@@ -133,7 +133,27 @@ module.exports = {
     "unicorn/empty-brace-spaces": "off",
     "unicorn/no-nested-ternary": "off",
     "unicorn/number-literal-case": "off",
-    "unicorn/template-indent": "off",
+    "unicorn/template-indent": [
+      'error',
```

Comment:
This is unacceptable, not everyone use this plugin, and you should not config settings for users in prettier congig.

Reply:
Yeah, this is not what I meant.

I meant that:

- The rule should: Be placed in this section with `0` instead of `"off"`: https://github.com/prettier/eslint-config-prettier/blob/f5530d3af4ce34ffb6402e11895bd18979835035/index.js#L7-L22
- We should look into if it’s feasible to make a validator for it: https://github.com/prettier/eslint-config-prettier/blob/f5530d3af4ce34ffb6402e11895bd18979835035/bin/validators.js
- We should document in the readme: https://github.com/prettier/eslint-config-prettier#special-rules

Reply:
Hmm I see

Reply:
> This is unacceptable, not everyone use this plugin, and you should not config settings for users in prettier congig.

I would honestly disagree just because the conflict in this rule is specifically in templates using the `html` tag, so one could imagine surgically turning *"off"* that part instead of shutting the whole thing down

Reply:
The readme of this project says at the top:

> Note that this config _only_ turns rules _off,_ so it only makes sense using it together with some other config.

We don’t want to change that.

Reply:
I see, will update the PR accordingly